src/main/kotlin/blackjack/domain/ScoreCalculator.kt

```diff
@@ -0,0 +1,27 @@
+package blackjack.domain
+
+object ScoreCalculator {
+    fun calculate(cards: List<Card>): Int {
+        val sumOfNotAceScore =
+            cards.filter { card -> card.isAce.not() }
```

Comment:
`filterNot` 확장 함수를 이용하도록 변경해볼 수도 있을 것 같아요 😉
```suggestion
            cards.filterNot { card -> card.isAce }
```

Reply:
오오.. 확장함수를 이렇게 활용하면 좋겠군요! 감사합니다